### ISSUE TITLE
Problem with NestedTernaryCheck

### ISSUE THREAD

**Issue by KTannenberg** [sevntu-checkstyle/sevntu.checkstyle#167]
NestedTernaryCheck have false-positive warnings:
For example in case of string concatenation:

```
return Joiner.on(" and ").join(Collections2.transform(columns, new Function<String, String>() {
    @Override
    public String apply(String column) {
        return alias1 != null ? dialect.quote(alias1, column) : dialect.quote(column) 
              + " = "
              + alias2 != null ? dialect.quote(alias2, column) : dialect.quote(column);
    }
}));
```

If ternary expressions are wrapped into parentheses to explicitly indicate priority of operations no warning is triggered.
